testsuite/MDAnalysisTests/topology/test_guessers.py

```diff
@@ -79,6 +79,10 @@ def test_guess_atom_element_singledigit(self):
     def test_guess_atom_element_1H(self):
         assert guessers.guess_atom_element('1H') == 'H'
         assert guessers.guess_atom_element('2H') == 'H'
+    
+    def test_guess_element_symbols(self):
```

Comment:
look up how `@pytest.parametrize` works (grep around in our tests and find one).  You should be able to write a list of `[(input1, expected1), (input2, expected2), ...]` which will make rattling off a lot of tests easy + expandable